styleguide/source/_patterns/01-molecules/09-search-results/_search-results.scss

```diff
@@ -0,0 +1,130 @@
+.search-result {
+  margin-bottom: $space;
+  width: 100%;
+  display: flex;
+  cursor: pointer;
+
+  &:active,
+  &:hover {
+    .search-result_link {
+      text-decoration: underline;
+    }
+
+    .search-result_left {
+      background-color: $black-15;
+    }
+  }
+}
+
+.search-result_label {
+  text-transform: uppercase;
+  color: $purple;
+  margin-bottom: 20px;
+
+  @include breakpoint($bp-small) {
+    margin-bottom: 25px;
+  }
+}
+
+.search-result_title {
+  @include type($font-serif, 18px, $font-weight-medium, 1.2222222222);
```

Comment:
the title font is looking a little bit small at mobile/tablet.

In the typography.scss file there are some preexisting type setups. This is spec'd as an "h4" (different from an h4 in the styleguide, i know its a bit confusing) which is 30/34 at desktop and tablet, and 20/24 at mobile. I tried to find its equivalent in the styleguide, but I couldn't find it.

You can see the original spec'd font sizes in this link https://drive.google.com/file/d/0B4-QX5V_ClpsLThhZ194cDVoMTg/view?usp=sharing